content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Pod Topology Spread Constraints
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+You can use _topology spread constraints_ to control how {{< glossary_tooltip text="Pods" term_id="Pod" >}} are spread across your cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Motivation
+
+In Kubernetes, directives related to "Affinity" control how Pods are
+scheduled - more packed or more scattered. But right now only limited options
+are offered: for `PodAffinity`, you can try to pack any number of Pods into qualifying
+topology domain(s); for `PodAntiAffinity`, only one Pod can be scheduled into a
+single topology domain.
+
+This is not an ideal situation if users want to distribute Pods evenly across different
+topology domains - to achieve high availability or cost-saving. And
+regular rolling upgrade or scaling out replicas can also be problematic.
+
+## Prerequisites
+
+### Enable Feature Gate
+
+Ensure the `EvenPodsSpread` feature gate is enabled (it is disabled by default
+in 1.16). See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling feature gates. The `EvenPodsSpread` feature gate must be enabled on apiserver **and** scheduler.
+
+### Apply Labels to Worker Nodes
+
+This feature requires the nodes to be labelled with topological key-value pairs via `kubectl label node <node_name> <key1=val1> <key2=val2> ...`. For example, you can verify using `--show-labels` flag:
```

Comment:
```suggestion
Topology spread constraints rely on node labels to identify the topology domain(s) that each Node is in. For example, a Node might have labels: `node=node1,zone=af-northeast-1b,region=africa-northeast`
```

In a Concept guide, the docs don't need to explain how to set and get Node labels.